src/constraints.jl

```diff
@@ -172,6 +172,22 @@ function delete(model::Model, con_ref::ConstraintRef{Model})
     MOI.delete(backend(model), index(con_ref))
 end
 
+"""
+    delete(model::Model, con_refs::Vector{ConstraintRef})
```

Comment:
The signature in the documentation should match the function signature (`Vector{<:ConstraintRef{Model}}` vs `Vector{ConstraintRef}`)

Reply:
I can do it. However, I think it is worth pointing that the single variable delete has signature:
`delete(model::Model, con_ref::ConstraintRef{Model})`
but the documentation comment is:
`delete(model::Model, con_ref::ConstraintRef)`
So I thought the preferred style was to drop the type parameters if they are little relevant.

Should I fix the documentation of the single variable method too?

Reply:
These are two different cases. No instance of `Vector{<:ConstraintRef{Model}}` will match to  `Vector{ConstraintRef}`, so this is plain incorrect. See the discussion [here](https://docs.julialang.org/en/v1/manual/types/index.html#Parametric-Composite-Types-1).

For the single-variable delete, any `ConstraintRef{Model}` is also a `ConstraintRef`, so this is broadening the scope of the method. I'm not sure how I feel about the documentation not matching the method signature, but this case is much less misleading.

Reply:
You are correct about the differences between the cases, I had not considered it that way. I will drop the the type parameters to keep conformance with how it is already done and because I have no opinion on this (nor I think this PR should bother about this). If you think this is not good you can change it at any moment.